lib/ASTGen/Sources/ASTGen/Decls.swift

```diff
@@ -575,11 +569,11 @@ extension ASTGenVisitor {
     return .createParsed(
       self.ctx,
       declContext: self.declContext,
-      importKeywordLoc: node.importKeyword.bridgedSourceLoc(in: self),
+      importKeywordLoc: self.bridgedSourceLoc(syntax: node.importKeyword),
       importKind: importKind,
-      importKindLoc: node.importKindSpecifier.bridgedSourceLoc(in: self),
+      importKindLoc: self.bridgedSourceLoc(syntax: node.importKindSpecifier),
       path: node.path.lazy.map {
-        $0.name.bridgedIdentifierAndSourceLoc(in: self) as BridgedIdentifierAndSourceLoc
+        self.bridgedIdentifierAndSourceLoc(token: $0.name) as BridgedIdentifierAndSourceLoc
```

Comment:
Didn't realize this was already the case, but the return type overloading here makes me sad, maybe we should rename `BridgedIdentifierAndSourceLoc` to `BridgedLocatedIdentifier` or something and rename this overload of the method too